IncludeToolbox/Commands/TryAndErrorRemoval_Project.cs

```diff
@@ -112,19 +103,39 @@ private bool ProcessNextFile()
                     continue;
 
                 impl.PerformTryAndErrorRemoval(document, settings);
-                ++fileIdx;
                 return true;
             }
-
-            projectItems = null;
             return false;
         }
 
+        static void RecursiveFindFilesInProject(ProjectItems items, ref Queue<ProjectItem> projectFiles)
+        {
+            var e = items.GetEnumerator();
+            while (e.MoveNext())
```

Comment:
Correct me if I'm wrong, but aren't you skipping the first file here?
You could just use a foreach loop (which is nothing else than a code generator that uses GetEnumerator anyways)